src/sympc/approximations/sigmoid.py

```diff
@@ -41,3 +44,31 @@ def sigmoid(tensor: MPCTensor, method: str = "exp") -> "MPCTensor":
             result += (tensor ** d) * weights[i + 1]
 
         return result
+
+    elif method == "chebyshev":
```

Comment:
Q: There was also this implementation in the CrypTen repo - maybe it was double-checking this implementation with theirs?

Reply:
https://github.com/facebookresearch/CrypTen/blob/eafda069e7fd19c91b6b08b678342019a95862f3/crypten/common/approximations.py#L326

Seems to be differently implemented :thinking: